### DISCUSSION TITLE
`Issues` — only for bug reports, not for FR?

### DISCUSSION THREAD

**Discussion by Korb** [readest/readest#25] [Q&A] [answered]
Currently, the Issues section has only one template - [for a bug report](https://github.com/chrox/readest/issues/new?assignees=&labels=type%3A+bug&projects=&template=report-bug.md&title=Ex%3A+In+DarkMode%2C+a+blank+square+appears+in+bottom+right+corner+while+scrolling). Am I correct in understanding that the developers of this project do not accept or consider **feature requests**?

**Comment by chrox** [comment, marked as answer]
Already added a template for feature request. And FR is always welcome.
